lib/cors/validate.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+'use strict'
+
+const validUrl = require('valid-url')
+
+/* ::
+import type {CorsConfiguration} from './read.js'
+*/
+
+const HELP = ', see documentation for information on how to configure cors.'
+
+function validateCors (
+  cors /* : CorsConfiguration */
+) /* : Promise<CorsConfiguration> */ {
+  if (!cors) {
+    return Promise.reject(new Error('Must specify cors configuration' + HELP))
+  }
+  if (!cors.origins || !Array.isArray(cors.origins) || !cors.origins.length) {
+    return Promise.reject(new Error('Must specify at least a single allowable origin in cors configuration' + HELP))
```

Comment:
You should be allowed to have either 0 origins, or 1 origin, due to the limitations we've discovered with AWS Lambda and/or Serverless.

Once that's been sorted out, we can allow multiple origins.